converters/toZigbee.js

```diff
@@ -3769,14 +3783,28 @@ const converters = {
                         );
                         state['color'] = {x: color.x, y: color.y};
                     } else if (color.hasOwnProperty('hue') && color.hasOwnProperty('saturation')) {
-                        const hsv = utils.gammaCorrectHSV(utils.correctHue(color.hue, meta), color.saturation, 100);
-                        extensionfieldsets.push(
-                            {
-                                'clstId': 768,
-                                'len': 13,
-                                'extField': [0, 0, (hsv.h % 360 * (65535 / 360)), (hsv.s * (2.54)), 0, 0, 0, 0],
-                            },
-                        );
+                        if (meta.mapped && meta.mapped.meta && meta.mapped.meta.enhancedHue === false) {
```

Comment:
I think we should not allow this since this behaviour is very implicit. Throwing an error with a message like: "hue/saturation not supported, use XY" makes more sense I think.